testdata/terraform/cloudflare_load_balancer_pool/test.tf

```diff
@@ -10,11 +10,11 @@ resource "cloudflare_load_balancer_pool" "terraform_managed_resource" {
   origins {
     address = "0.0.0.0"
     enabled = true
-    name    = "app-server-1"
-    weight  = 1
     header {
       header = "Host"
       values = ["example.com"]
     }
+    name   = "app-server-1"
+    weight = 0.56
```

Comment:
This was rather interesting. AFAICT we were actually generating this incorrectly before. 
https://github.com/cloudflare/cf-terraforming/blob/master/testdata/cloudflare/cloudflare_load_balancer_pool.yaml#L37

Reply:
nice find!